docs/tutorials/gettingstartedexample/gse-gen-common.md

```diff
@@ -38,7 +38,7 @@ This tutorial shows you how to configure the **FPGA Display Parameters** to chan
     |---|---|---|
     | Resource | RIO Device | [System Specific] |
     | Resource | Bitfile Path | [Refer to Bitfile Path in the PXIe-148X Generation GSE Help](../../reference/gettingstartedexample/gse-gen-help.md#table-of-pxie-148x-generation-bitfiles) |
-    | Resource | TDMS File Directory| A directory path containing a file named <font face = "courier new">SI0_LLP_Packets.tdms</font> |
+    | Resource | TDMS File Directory| A directory path to TDMS files containing LLP Packet data for serial output channels (e.g. SI0_LLP_Packets.tdms) |
```

Comment:
Oh this is a such a nitpick on my part, but I thought you would leave '<font face = "courier new">' around the file name when changed.

Reply:
Done